### ISSUE TITLE
Contents insurance 

### ISSUE THREAD

**Issue by iteles** [dwyl/hq#81]
- [ ] Research
- [ ] Get!

**Comment by markwilliamfirth** [comment]
@iteles do we want this?

**Comment by markwilliamfirth** [comment]
Links to #256

**Comment by iteles** [comment]
Let's look into how much it costs, but I'm pretty sure we will!

**Comment by markwilliamfirth** [comment]
@iteles need these answers to get quote

1. Office contents
How much would it cost to replace all property permanently located at your office?

2. Portable equipment
How much would it cost to replace all other property owned by the business?
This should include things like laptops, mobile phones, cameras, etc.

3. Increased cost of working
This covers additional expenses needed to keep your business up and running following damage to your property or the premises you work in (such as renting temporary alternative premises).

4. Money and assault
Do you require cover for cash? [up to 2.5k limit for cover]

**Comment by iteles** [comment]
@markwilliamfirth Can I suggest we start with an inventory of everything in QS?

I'll need to do the same for portable equipment at HQ.

**Comment by markwilliamfirth** [comment]
Closed in favour of #256

**Comment by rub1e** [comment]
Can someone tell me where the inventory of everything in QS/FH which Ines requested one year ago is saved please, _Rubie asked, full of wide-eyed optimism, and holding his breath..._

**Comment by markwilliamfirth** [comment]
Hi @rub1e :wave: I hope you are well! This issue was blocked by https://github.com/dwyl/focus-hub/issues/37

I hope you don't mind, but can I ask a quick favour? Would it be possible to try to not to comment on closed issues I'm subscribed to (instead a new issue might be better?) and tagging me as below:

<img width="793" alt="screen shot 2018-02-02 at 13 32 26" src="https://user-images.githubusercontent.com/5732209/35731238-8c1eb7ac-081d-11e8-9bc9-e8309016600f.png">

I have already unsubscribed from a large number of issues, but when I am tagged or comments are made on closed issues I get unnecessary notifications to my personal GitHub account. Anything you do to help would be greatly appreciated. Thank you for your help, Rudie!
